features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
+    (org.nil? || org.empty?) ? (nil) : (org = org.strip())
+    if ((org.nil? || org.empty?) || (chart.nil? || chart.empty?))
```

Comment:
Didn't lines 6-7 just ensure that `org` and `chart` are either nil or non-empty? If so, simplify this to just check for nils.